package.json

```diff
@@ -89,6 +89,18 @@
         "when": "editorTextFocus && editorLangId == 'azcli'"
       }
     ],
+    "configuration": {
+      "type": "object",
+      "title": "Azure CLI Tools Configuration",
+      "properties": {
+        "ms-azurecli.showResponseInDifferentTab": {
```

Comment:
`"azureCLI.showResultInNewEditor"` will render the prefix nicer in the settings editor and 'new editor' is more general than 'new tab' (you can disable tabs).

Reply:
Changed. I never liked how the prefix was being rendered but hadn't dug into how to fix it yet - thanks!